### DISCUSSION TITLE
when ask ovos what time it is, it answers in a variety of ways.

### DISCUSSION THREAD

**Discussion by geetee24** [OpenVoiceOS/ovos-buildroot#260] [Q&A] [answered]
what file do i edit (path to it please) to make ovos always answer "the time is....."

thank you

**Comment by JarbasAl** [comment]
No path to edit, uninstall the official date time skill and install your own fork of the skill with your changes

**Comment by geetee24** [comment]
What is the name of the official daytime skill so I can find it?

**Reply by mikejgray** [reply]
https://github.com/OpenVoiceOS/skill-ovos-date-time/

**Comment by JarbasAl** [comment, marked as answer]
just added docs for this

https://openvoiceos.github.io/ovos-technical-manual/user_res_files/

it has not been very well tested yet, let us know how it goes if you try it

for any interested devs the relevant code is here https://github.com/OpenVoiceOS/OVOS-workshop/blob/dev/ovos_workshop/resource_files.py#L204
